spec/factories/outgoing_messages.rb

```diff
@@ -22,6 +22,7 @@
   factory :outgoing_message do
     info_request
     prominence { 'normal' }
+    last_sent_at { 2.weeks.ago }
```

Comment:
This seems unrelated to the fact that events aren't getting created?

Might be worth separating out as its own commit, or at least describing why we also need this in this commit msg.

Reply:
Yeah just noticed this and trying to remember why this is necessary. Will separate out in to its own commit.